src/search/_base.scss

```diff
@@ -49,4 +49,72 @@
 			margin-right: auto;
 		}
 	}
+
+	//new styles from the update
+	h2 {
+		font-family: "Noto sans", sans-serif;
+		font-size: 20px;
+		line-height: 34px;
+		padding-bottom: 8px;
+		padding-top: 8px;
```

Comment:
Is there a rational to style the heading differently from the default? like using "Noto sans" instead of "Lato",....

Heading need to be differentiated from normal text

Reply:
I've changed the 'h2' to simply a paragraph tag, since on further research, the number of results isn't really a heading, and isn't used as such by other search engines like Google or Bing.

I've added a 'results-count' class specifically for this one, and faded it a bit. We should also probably remove all the padding around it, but can't seem to...

I've changed the search result link to an h2 class="h3", and we can just use the standard Canada.ca h2 styling, which is lato 24px, if I saw correctly.

Hope that works!